ee/localserver/krypto-ec-middleware.go

```diff
@@ -92,6 +93,9 @@ func (e *kryptoEcMiddleware) Wrap(next http.Handler) http.Handler {
 		}
 
 		v := url.Values{}
+		for key, val := range cmdReq.UrlParameters {
+			v.Add(key, val)
+		}
```

Comment:
This is basically supporting GET parameters. Do you think it makes sense to look at POST data instead? queries can get pretty big...

Reply:
I don't know, adding url parameters just fit so neatly =) if we introduce posts, now the middleware has to make a decision about weather the next endpoint it a post or a get. I will add some tests with huge url params and see if it effects anything.

Reply:
Does it? Can we hint it from the cmd? 

I guess if the data fits in a GET, then maybe we can keep ignoring this. After all, at the point in time we need a POST, we probably need it to localserver as well

Reply:
After some testing, I did start getting errors when url params got too big. Since everything comming _into_ local server is a GET request, we will have some limitations there.

However, we can switch the internal routing to all POSTs, then we don't have to make decisions in the middle ware about weather it's a GET or POST and we just add a `body []byte` filed to the `cmdReq`

... what say you @directionless ?

Reply:
I agree we'll need to handle the incoming data as POST. I think localserver can probably just handle both, I remember I did something obvious to limit it to GET.

I don't think I have an opinion about whether we push it all through post internally or not. 

We need to be consistent in the priority order of where `box` comes from, but aside from that, I think we can kinda just do whatever.

Reply:
updated to handle both incoming get and post data, un sure of what post body should look like, see https://github.com/kolide/launcher/pull/1030#discussion_r1108773392